lighthouse-core/audits/seo/canonical.js

```diff
@@ -10,6 +10,31 @@ const LinkHeader = require('http-link-header');
 const URL = require('../../lib/url-shim');
 const MainResource = require('../../computed/main-resource.js');
 const LINK_HEADER = 'link';
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on a page's rel=canonical link. This descriptive title is shown to users when the rel=canonical link is valid. */
+  title: 'Document has a valid `rel=canonical`',
+  /** Title of a Lighthouse audit that provides detail on a page's rel=canonical link. This imperative title is shown to users when the rel=canonical link is invalid and should be fixed. */
```

Comment:
it's a descriptive title :)

Reply:
now you know we are just copying these things from other examples :) i've done this same thing lol